portal-api/src/scheduler/jobs/index.js

```diff
@@ -1,6 +1,7 @@
+const sendReportDueEmailsJob = require('./send-report-due-emails-job');
 const sendReportingPeriodStartEmailsJob = require('./send-reporting-period-start-emails-job');
 
 /**
  * @type {typeof import('../types/scheduler-job').SchedulerJob[]}
  */
-module.exports = [sendReportingPeriodStartEmailsJob];
+module.exports = [sendReportDueEmailsJob, sendReportingPeriodStartEmailsJob];
```

Comment:
Maybe put the elements of the array onto new lines to match the formatting elsewhere

Reply:
I didn't realise prettier formatted it this way - if it does then no issue :)